onboarding/main.go

```diff
@@ -27,9 +26,9 @@ const (
 )
 
 func main() {
-	clientset, err := newClient()
+	cl, err := newClient()
 	if err != nil {
-		klog.Errorf("failed to create clientset: %v", err)
+		klog.Errorf("failed to create client: %v", err)
 		os.Exit(1)
```

Comment:
You have access to `klog.Exit` and `klog.Exitf` use them instead of `os.Exit`.
Please apply the change to all instances of `os.Exist` in this file.